### PR TITLE
Bug fix/export anisotropic annotations

### PR DESCRIPTION
**Pull request by jpdefrutos**
Related to issue #53 

The original image aspect ratio is used to scale the spline segmentation coordinates to match the original image resolution.

In addition, the field offset in MHD metada headers has been exposed to correctly align the segmentations.

### CODE REVIEW

File: exporters/spline_segmentation_exporter.py [hunk 1]
```diff
@@ -258,6 +365,9 @@ def save_segmentation(self, frame, image_size, filename, spacing, json_annotatio
         else:
             segmentation_mhd = MetaImage(data=segmentation)
             segmentation_mhd.set_attribute('ImageQuality', frame.image_annotation.image_quality)
+            if 'image_mhd' in locals():
+                segmentation_mhd.set_attribute('FrameType', image_mhd.get_metaimage_type())
```
Comment (smistad) [inline exporters/spline_segmentation_exporter.py:369]: What is the purpose of setting FrameType here to metaimage data type?
This is already set using the standard ElementType field: f.write('ElementType = ' + self.get_metaimage_type() + '\n')
Also the FrameType attribute is used for something else in cardiac applications.

File: exporters/spline_segmentation_exporter.py [hunk 2]
```diff
@@ -163,68 +163,72 @@ def get_object_segmentation(self, image_size, frame):
...
+                #         xy.append(previous_y)
+                #
+                #         segmentation[y, x] = counter
+                #
+                #     xy_new = [xy[j:j + 2] for j in range(0, len(xy), 2)]
+                #
+                #     if i == max_index-1 and xy_new_temp != xy_new:
+                #         coordinates.append(xy_new)
+                #         coordinates.append(a.label)
+                #         xy_new_temp = xy_new
```
Comment (smistad) [inline exporters/spline_segmentation_exporter.py:231]: If this code is not used anymore it can be removed.

File: boundingbox/static/boundingbox/boundingbox.js
Comment (smistad) [inline boundingbox/static/boundingbox/boundingbox.js:1]: This seems like it should not be part of this PR.
Also the functionality here doesn't work. It crashes with the error: Uncaught TypeError: boxes_to_copy is undefined

**Review by smistad — COMMENTED** [review]
Export functionality of anisotropic data in spline segmentation task seems to work fine. Nice job.
However I have some questions. Also there seems to be some other functionality added here (copying bounding boxes) which does not work.

File: annotationweb/static/annotationweb/annotationweb.js
[the remark this answers is not in the document]
Comment (jpdefrutos) [reply annotationweb/static/annotationweb/annotationweb.js:1]: Removed the unnecessary check (see 95f9f183906a626e493e5a8783e4614cb1b03cd7)

File: exporters/spline_segmentation_exporter.py (hunk 1 above)
Comment (jpdefrutos) [reply exporters/spline_segmentation_exporter.py]: Removed (see 6952e65ab1e4e5e0965844445bf2f255c728c2f8)

File: exporters/spline_segmentation_exporter.py (hunk 2 above)
Comment (jpdefrutos) [reply exporters/spline_segmentation_exporter.py]: Forgot to remove it. Fixed in 6952e65ab1e4e5e0965844445bf2f255c728c2f8

File: boundingbox/templates/boundingbox/process_image.html
Comment (smistad) [inline boundingbox/templates/boundingbox/process_image.html:1]: Since you removed the copyToNext javascript function, you should probably remove this from the PR as well.

**Review by smistad — COMMENTED** [review]
Looks good, but you since you removed the copyToNext javascript function from this PR you should probably remove the button which calls this function as well.

File: boundingbox/templates/boundingbox/process_image.html
Comment (jpdefrutos) [reply boundingbox/templates/boundingbox/process_image.html:1]: Solved. See commit 34481a6
